### PR TITLE
public editor update

### PR DESCRIPTION
**Pull request by MSlootwegVIKTOR**
This pull request serves to update the current app to a VIKTOR public editor.

Take note: Once the pull request is approved, will the README be updated with the relevant images/gifs.

### CODE REVIEW

**Comment by puijterwaal** [comment]
Can you move the cover image to `resources/cover_image.png` ? This will be standardized for all sample apps repos beginning of jan

File: README.md
Comment (puijterwaal) [inline README.md:23]: Maybe it would be nice to make it explicit in the readme that this is an editor-only app type

File: app/cpt_file/controller.py
Comment (puijterwaal) [inline app/cpt_file/controller.py:25]: Inconsistent with how we tend to have it setup (one import per line)

File: app/cpt_file/controller.py
Comment (puijterwaal) [inline app/cpt_file/controller.py]: Will you wait for that to be released to merge to main? otherwise it might be better to make an issue instead of leaving it as a todo

File: app/cpt_file/controller.py
Comment (puijterwaal) [inline app/cpt_file/controller.py:113]: I think it used to be more readable before, on the left.

File: app/cpt_file/parametrization.py
Comment (puijterwaal) [inline app/cpt_file/parametrization.py:72]: Why are the tabs missing?

File: app/cpt_file/soil_layout_conversion_functions.py
Comment (puijterwaal) [inline app/cpt_file/soil_layout_conversion_functions.py]: I personally like it better `meter_to_mm` as it can be difficult to distinguish `m_to_mm` from `m_to_mm` and this can lead to mistakes

File: app/cpt_file/soil_layout_conversion_functions.py
Comment (puijterwaal) [inline app/cpt_file/soil_layout_conversion_functions.py:68]: I think you have the black line length set really short

File: CHANGELOG.md
Comment (puijterwaal) [inline CHANGELOG.md:6]: This should be a new tag, instead of added to the previous?

File: requirements.txt
Comment (puijterwaal) [inline requirements.txt:2]: nice! adding it to the checklist for all of the sample apps

File: CHANGELOG.md
Comment (puijterwaal) [inline CHANGELOG.md]: Changed upload from File-entity (with params from file) to using the FileField instead

File: README.md
Comment (puijterwaal) [inline README.md]: remove `<basic one line intro here>`

File: README.md
Comment (puijterwaal) [inline README.md:20]: rename this file  (see comment about standardised cover_image)

File: README.md
Comment (puijterwaal) [inline README.md:3]: Should we also update the repo name?

File: README.md
Comment (MSlootwegVIKTOR) [reply README.md:23]: Good idea. I've reset the App structure heading with this as info added.

File: app/cpt_file/controller.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/controller.py:25]: Hmmm... I've used `black` to quickly fix any formatting mistakes. Are setting this format manually, or using another formatting/linting package?

File: app/cpt_file/controller.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/controller.py]: I'm waiting for the release before merging.

File: app/cpt_file/controller.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/controller.py:113]: I agree, `black` changed this. I'll be using line-length of 119 from now onwards...

File: app/cpt_file/parametrization.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/parametrization.py:72]: The tab has been replaced with steps. Only one tab was used. Is that what you mean?

File: app/cpt_file/soil_layout_conversion_functions.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/soil_layout_conversion_functions.py]: Reverted.

File: app/cpt_file/soil_layout_conversion_functions.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/soil_layout_conversion_functions.py:68]: I think so too, although I am using the default.

File: CHANGELOG.md
Comment (MSlootwegVIKTOR) [reply CHANGELOG.md:6]: We have not yet released a new tag (I assume?), therefore I've added it to the current unreleased tag.

File: README.md
Comment (MSlootwegVIKTOR) [reply README.md:3]: I was also considering this... I think it would be better...

File: app/cpt_file/controller.py
Comment (puijterwaal) [reply app/cpt_file/controller.py]: clear!

File: app/cpt_file/parametrization.py
Comment (puijterwaal) [reply app/cpt_file/parametrization.py:72]: oh i mean "spaces"? This text is fully lined out with the left margin. I don't really mind, it can be nicer to have the space, I'm just not used to it ;p

File: README.md
Comment (puijterwaal) [reply README.md:3]: I agree! Do you have the rights? Else I can do it for you

File: app/cpt_file/parametrization.py
Comment (puijterwaal) [inline app/cpt_file/parametrization.py]: Being really nitpicky, "change classification table" of "use custom table" is maybe clearer?

File: app/cpt_file/parametrization.py
Comment (puijterwaal) [inline app/cpt_file/parametrization.py]: I think this line break is not really needed?

File: app/cpt_file/controller.py
Comment (puijterwaal) [inline app/cpt_file/controller.py]: Based on our discussion. I think this is a good solution!

**Review by puijterwaal — APPROVED** [review]

File: CHANGELOG.md
Comment (puijterwaal) [reply CHANGELOG.md:6]: Mistake on my side then!

File: app/cpt_file/parametrization.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/parametrization.py:72]: Oh I understand what you mean. The tab can produce in some instances weird formatting, so to avoid this, I have done this consistently on all text.

File: README.md
Comment (MSlootwegVIKTOR) [reply README.md:20]: The cover image is, although slightly, different from the cpt_visualization.png image.

File: README.md
Comment (MSlootwegVIKTOR) [reply README.md:3]: Could you do this please?

File: app/cpt_file/parametrization.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/parametrization.py]: Haha agree, was debating with myself too much, so just pushed it to see what you would think 😄

File: app/cpt_file/parametrization.py
Comment (MSlootwegVIKTOR) [reply app/cpt_file/parametrization.py]: Removed.

**Pull request merged**
